### PR TITLE
set up a webpage

### PR DESCRIPTION
**Pull request by gurtatiLND**

### CODE REVIEW

**Comment by gurtatiLND** [comment]
I have just set up a new webpage. Please review and let me know if any changes are needed.

File: users/gurtatiLND/website/webpage.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        </header>
+        <main>
+            <section class="home">Home</section>
+            <section id="about" class="about sec-pad">About</section>
+            <section id="projects" class="projects sec-pad">Projects</section>
+            <section id="contact" class="contact sec-pad">Contact</section>
+        </main>
+        <footer class="main-footer">
+            <p>
+                @Copyright 2024. Made by <a href="https://github.com/gurtatiLND">Tatiana Gurova</a>.
```
Comment (CringleySDays) [inline users/gurtatiLND/website/webpage.html:21]: To replicate the copyright symbol, I'd recommend trying to look for HTML character entity reference. 

```html
<p> &copy; Copyright 2024.... </p>
```

Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:21]: oh thank you! will do this

File: users/gurtatiLND/website/webpage.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="'en">
```
Comment (CringleySDays) [inline users/gurtatiLND/website/webpage.html:2]: You made an error here in accidentally adding extra string

**Comment by gurtatiLND** [comment]
@CringleySDays I have addressed the issues you mentioned in your review. Thank you for your feedback!

**Pull request merged**